assets/js/app.js

```diff
@@ -0,0 +1,320 @@
+            button.style.backgroundColor = `grey`;
+            button.innerHTML = `add`;
+            toggledArray.splice(toggledArray.indexOf(button.id), 1);
+        } else { 
+            if(toggledArray.length < 5){
+                button.style.backgroundColor = 'blue';
+                button.innerHTML = `remove`;
+                toggledArray.push(button.id);
+            } else {
+                const triggeringButtonId = button.id;
+                // jump window
+                const jumpWindow = document.createElement('div');
+                jumpWindow.className = 'jumpWindow';
+                // jump window content
+                jumpWindow.innerHTML = `
+                    <p>only 5 coins can be viewed at a time. </br> Please select a coin to replace:</p>
+                    <button id="${toggledArray[0]}" class="jumpWindowButtons">X</button> <span>${toggledArray[0]}</span>
+                    </br></br>
+                    <button id="${toggledArray[1]}" class="jumpWindowButtons">X</button> <span>${toggledArray[1]}</span>
+                    </br></br>
+                    <button id="${toggledArray[2]}" class="jumpWindowButtons">X</button> <span>${toggledArray[2]}</span>
+                    </br></br>
+                    <button id="${toggledArray[3]}" class="jumpWindowButtons">X</button> <span>${toggledArray[3]}</span>
+                    </br></br>
+                    <button id="${toggledArray[4]}" class="jumpWindowButtons">X</button> <span>${toggledArray[4]}</span>
+                    </br></br>
+                    <button class="closeJumpWindow">Cancel</button>
+                `;
+                document.body.appendChild(jumpWindow);
+                // change selected buttons AND close jump window
+                document.querySelectorAll('.jumpWindow button:not(.closeJumpWindow)').forEach(button=> button.addEventListener('click', function() {
+                    const button = document.getElementById(this.id)
+                    const triggeringButton = document.getElementById(triggeringButtonId);
+                    button.style.backgroundColor = `grey`;
+                    button.innerHTML = `add`;
+                    toggledArray.splice(toggledArray.indexOf(button.id), 1);
+                    document.body.removeChild(jumpWindow);
+                    toggledArray.push(triggeringButtonId);
+                    triggeringButton.style.backgroundColor = 'blue';
+                    triggeringButton.innerHTML = `remove`;
+                }));
+                // close jump window without changing selected buttons
+                document.querySelector('.closeJumpWindow').addEventListener('click', () => {
+                    document.body.removeChild(jumpWindow);
+                });
+            }
+        }
+    }));
+
+    // generate more info on coins + loading spinner + cache check
+    document.querySelectorAll('.info-button').forEach(button => button.addEventListener('click', async () => {
```

Comment:
wasn't explicitly mentioned, but it was expected to hide any other `infoContainer` when opening a new one